pkgs/applications/editors/kakoune/plugins/overrides.nix

```diff
@@ -131,6 +132,41 @@ declare-option -hidden str ansi_filter %{'"$out"'/bin/kak-ansi-filter}
     '';
   });
 
+  hop-kak = rustPlatform.buildRustPackage rec {
```

Comment:
This package may not be suitable here?

Reply:
where do you think it should go? this seems to be the file for kakoune plugins that have some build that is not just fetching.